NOTICE.txt

```diff
@@ -786,8 +786,8 @@ Apache License 2.0
 
 --------------------------------------------------------------------
 Dependency: github.com/elastic/go-sysinfo
-Version: v1.0.2
-Revision: 06c1f463545498d8f4b378d4dcf3171794c28537
+Version: v1.1.0
```

Comment:
Take a look to sysinfo changelog and if it fixes things that would affect Beats, add it to the changelog.

Reply:
Yah, it was pretty much just this.